modules/topicsFpdModule.js

```diff
@@ -251,6 +260,25 @@ function loadTopicsForBidders() {
         setLoadedIframeURL(new URL(iframeURL).origin);
         iframeURL && window.document.documentElement.appendChild(ifrm);
       }
+
+      if (bidder && fetchUrl) {
+        let storedSegments = new Map(safeJSONParse(coreStorage.getDataFromLocalStorage(topicStorageName)));
+        const bidderLsEntry = storedSegments.get(bidder);
+
+        if (!bidderLsEntry || (bidderLsEntry && isCachedDataExpired(bidderLsEntry[lastUpdated], fetchRate || DEFAULT_FETCH_RATE_IN_DAYS))) {
+          window.fetch(`${fetchUrl}?bidder=${bidder}`, {browsingTopics: true})
```

Comment:
Can we also send topics send topics in the bid request using the header method?

Reply:
yes but that is blocked by or part of https://github.com/prebid/Prebid.js/issues/9980 ; your PR would be welcomed

Reply:
My understanding is the xhr exception will live as long as gam needs it and not any longer